lib/DDG/Meta/Data.pm

```diff
@@ -122,6 +125,26 @@ sub get_ia {
     return $m;
 }
 
+sub _satisfy {
+	my ($ia, $by, $lookup) = @_;
+	return $by->($ia, $lookup) if ref $by eq 'CODE';
+	my $pby = $ia->{$by};
+	ref $lookup eq 'CODE' ? $lookup->($pby) : $pby eq $lookup;
+}
+
+# get_ias(repo => 'goodies', dev_milestone => 'live'...)
+# Lookups combine as an AND operation.
+sub get_ias {
+	my (@lookups) = @_;
+	my @results = @ia_container;
+	foreach (pairs @lookups) {
```

Comment:
Still not quite following the `pairs` argument.  I'll just have to add some debugging.

There are mods out there that do that kind of logic.  Can't think of one off the top of my head.  I could very well see your example being taken in the sense of "id in (id1, id2, ...)"